### PR TITLE
Add Rails 7.1 to the test matrix

### PR DESCRIPTION
**Pull request by jcoyne**

### CODE REVIEW

**Comment by jcoyne** [comment]
This seems to be dependent on #2998

File: .github/workflows/ruby.yml [hunk 1]
```diff
@@ -37,11 +37,15 @@ jobs:
             blacklight_version: 'latest'
             bootstrap_version: ~> 4.0
             additional_engine_cart_rails_options: -a propshaft
+          - rails_version: '7.1.2'
```
Comment (taylor-steve) [inline .github/workflows/ruby.yml:40]: Curious if there's a reason to test on 7.1.2 or if we can jump to 7.1.3.4 now?

Comment (corylown) [reply .github/workflows/ruby.yml:40]: I think we can.

**Comment by taylor-steve** [comment]
The SKIP_TRANSLATION change is going to be nice to have, thanks.

**Review by taylor-steve — APPROVED** [review]

**Pull request merged**